x/hard/keeper/withdraw.go

```diff
@@ -9,12 +9,6 @@ import (
 
 // Withdraw returns some or all of a deposit back to original depositor
 func (k Keeper) Withdraw(ctx sdk.Context, depositor sdk.AccAddress, coins sdk.Coins) error {
-	// Get current stored LTV based on stored borrows/deposits
-	prevLtv, err := k.GetStoreLTV(ctx, depositor)
-	if err != nil {
-		return err
-	}
-
 	deposit, found := k.GetDeposit(ctx, depositor)
 	if !found {
 		return sdkerrors.Wrapf(types.ErrDepositNotFound, "no deposit found for %s", depositor)
```

Comment:
Ditto. Since we call `k.SyncBorrowInterest(ctx, depositor)`, we should call `k.BeforeBorrowModified(ctx, borrow)` if one exists.

Reply:
Fixed in https://github.com/Kava-Labs/kava/pull/800/commits/0d9bcf911e3cf6f625c7e72b40023d811679b401